<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>555</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1245</width>
    <height>180</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Lilyplayer</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QGraphicsView" name="music_sheet">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>1024</width>
        <height>600</height>
       </size>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>160</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="Playsubsequence">
          <property name="text">
           <string>Play sub sequence</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="startMeasureLabel">
            <property name="text">
             <string>start measure</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="start_measure"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="stopMeasureLabel">
            <property name="text">
             <string>stop measure</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="stop_measure"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="replay">
        <property name="text">
         <string>Play whole song</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="PlayPauseButton">
        <property name="text">
         <string>Play / Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>260</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QGraphicsView" name="keyboard">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1198</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1198</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>BlankCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1245</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuInput">
    <property name="title">
     <string>&amp;Input</string>
    </property>
    <addaction name="actionF_r_Elise_Beethoven"/>
    <addaction name="actionPrelidium_1_Bach"/>
    <addaction name="actionEtude_As_dur__Chopin"/>
    <addaction name="actionFile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInput"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open a midi file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSelect_file">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select file</string>
   </property>
   <property name="toolTip">
    <string>Open a midi file</string>
   </property>
  </action>
  <action name="actionF_r_Elise_Beethoven">
   <property name="text">
    <string>FÃ¼r Elise - Beethoven</string>
   </property>
  </action>
  <action name="actionPrelidium_1_Bach">
   <property name="text">
    <string>Prelidium 1 (BWV 846) - Bach</string>
   </property>
  </action>
  <action name="actionEtude_As_dur__Chopin">
   <property name="text">
    <string>Etude As dur - Chopin</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>Select file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionF_r_Elise_Beethoven</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>play_fuer_Elise__Beethoven()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEtude_As_dur__Chopin</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>play_Etude_As_dur__Chopin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrelidium_1_Bach</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>play_Prelidium_1__Bach()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlayPauseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>toggle_play_pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>849</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Playsubsequence</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sub_sequence_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replay</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>replay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
